script.js

```diff
@@ -0,0 +1,109 @@
+var REGION_NAME = "Impostor"
+var SERVER_PORT = 22023;
```

Comment:
Could you make it so you can generate with port? Among us game now supports different ports and the server will too.
I was thinking something like it defaults to 22023 but you can enter a custom one.